docs/Reference-for-template.json.md

```diff
@@ -197,6 +197,50 @@ String optional parameter, replaces TargetFrameworkOverride:
 }
 ```
 
+#### Multichoice symbols specifics
+Multichoice symbols have similar behavior and usage scenarios as [C# Flag enums](https://docs.microsoft.com/en-us/dotnet/api/system.flagsattribute) - they express a range of possible values (not a single value - unlike the plain [choice symbol](#choice-sample))
+
+##### Example definition of multichoice symbol:
+
+```json
+  "symbols": {
+    "Framework": {
+      "type": "parameter",
+      "description": "The target framework for the project.",
+      "datatype": "choice",
+      "allowMultiple": true,  // multichoice indicator
+      "choices": [
+        {
+          "choice": "netcoreapp3.1",
+          "description": "Target netcoreapp3.1"
+        },
+        {
+          "choice": "netstandard2.1",
+          "description": "Target netstandard2.1"
+        },
+        {
+          "choice": "netstandard2.0",
+          "description": "Target netstandard2.0"
+        }
+      ],
+      "defaultValue": "netstandard2.0|netstandard2.1"
```

Comment:
is pipe configurable here?

It's ok if it's not, just asking. It would be good to document it if pipe delimiter is predefined.

Reply:
I don't think configurability is needed, it would be good to document it though.